spec/facility_spec.rb

```diff
@@ -2,25 +2,192 @@
 
 RSpec.describe Facility do
   before(:each) do
-    @facility = Facility.new({name: 'Albany DMV Office', address: '2242 Santiam Hwy SE Albany OR 97321', phone: '541-967-2014' })
+    @facility_1 = Facility.new({
+      name: 'Albany DMV Office', 
+      address: '2242 Santiam Hwy SE Albany OR 97321', 
+      phone: '541-967-2014', 
+      daily_hours: '9-5',
+      holidays: 'All of them'
+    })
+    @facility_2 = Facility.new({
+      name: 'Ashland DMV Office', 
+      address: '600 Tolman Creek Rd Ashland OR 97520', 
+      phone: '541-776-6092' 
+    })
+    @cruz = Vehicle.new({
+      vin: '123456789abcdefgh', 
+      year: 2012, 
+      make: 'Chevrolet', 
+      model: 'Cruz', 
+      engine: :ice
+    })
+    @bolt = Vehicle.new({
+      vin: '987654321abcdefgh', 
+      year: 2019, 
+      make: 'Chevrolet', 
+      model: 'Bolt', 
+      engine: :ev
+    })
+    @camaro = Vehicle.new({
+      vin: '1a2b3c4d5e6f', 
+      year: 1969, 
+      make: 'Chevrolet', 
+      model: 'Camaro', 
+      engine: :ice
+    })
+    @registrant_1 = Registrant.new('Bruce', 18, true)
+    @registrant_2 = Registrant.new('Penny', 16)
+    @registrant_3 = Registrant.new('Tucker', 15)
   end
+
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('Albany DMV Office')
-      expect(@facility.address).to eq('2242 Santiam Hwy SE Albany OR 97321')
-      expect(@facility.phone).to eq('541-967-2014')
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('Albany DMV Office')
+      expect(@facility_1.address).to eq('2242 Santiam Hwy SE Albany OR 97321')
+      expect(@facility_1.phone).to eq('541-967-2014')
+      expect(@facility_1.services).to eq([])
+      expect(@facility_1.daily_hours).to eq('9-5')
+      expect(@facility_1.holidays).to eq('All of them')
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
```

Comment:
Great job testing the initial value and then testing again after making changes.